modules/deploy/pages/deployment-option/cloud/configure-privatelink-private-service-connect-in-cloud-ui.adoc

```diff
@@ -0,0 +1,88 @@
+= Configure PrivateLink and Private Service Connect in the Cloud UI
+:description: Set up AWS PrivateLink and GCP Private Service Connect in the Redpanda Cloud UI.
+:page-cloud: true
+
+include::shared:partial$feature-flag.adoc[]
+
+NOTE: This guide is for configuring AWS PrivateLink and GCP Private Service Connect using the Redpanda Cloud UI. See xref:deploy:deployment-option/cloud/aws-privatelink.adoc[Configure AWS PrivateLink for Redpanda Cloud] or xref:deploy:deployment-option/cloud/gcp-private-service-connect.adoc[Configure GCP Private Service Connect for BYOC] if you want to set up these services using the API.
+
+The Redpanda AWS PrivateLink and GCP Private Service Connect services provide secure access to Redpanda Cloud from your own VPC. Traffic over PrivateLink and Private Service Connect do not go through the public internet because these connections are treated as their own private AWS and GCP services. While your VPC has access to the Redpanda VPC, Redpanda cannot access your VPC.
+
+Consider using the endpoint services if you have multiple VPCs and could benefit from a more simplified approach to network management:
+
+* PrivateLink and Private Service Connect allow overlapping CIDR ranges in VPC networks.
+* PrivateLink and Private Service Connect do not limit the number of VPC connections.
+* You control which GCP projects or AWS resources are allowed to connect to the endpoint services.
+
+== Requirements
+
+[tabs]
+====
+AWS::
++
+* Your Redpanda cluster and VPC must be in the same region.
+* Use the https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html[AWS CLI] to create a new client VPC or modify an existing one to use the PrivateLink endpoint.
+GCP::
++
+* Use https://cloud.google.com/sdk/docs/install[gcloud^] to create the consumer-side resources, such as a VPC and forwarding rule, or modify an existing one to use the Private Service Connect service attachment created for your cluster.
```

Comment:
@0x5d does the [item](https://github.com/redpanda-data/docs/pull/516/files#diff-804f88d25737f50d4a1cef980e1c15f01b3b6390eb2ef979d0409907f972ccecR23) above for AWS regarding the RP cluster and VPC being in the same region also apply for PSC?

Reply:
"VPC" is kind of ambiguous here. If it refers to the Redpanda VPC, then yes, it must be in the same region. If it refers to a VPC where consumer resources will be deployed (i.e. Redpanda clients), then it must be in the same region unless Global Access is enabled for PSC (this knob is only available through the API right now)

Reply:
@0x5d I'll add the same region requirement to this doc for now, and specify that it's the client VPC.